docs/manual/dynamicinterpreterload.md

```diff
@@ -0,0 +1,130 @@
+<center><img src="../assets/themes/zeppelin/img/docs-img/zeppelin_user.png" height="85%" width="85%"></center>
+
+## Load & Unload Interpreters Using REST API
+
+### 1. Load 
+You can **load** interpreters located in Maven repository using REST API, like this:
+
+( Maybe, you are unfamiliar with `[interpreter_group_name]` or `[interpreter_name]`. If so, please checkout [Interpreters in Zeppelin](../manual/interpreter.html) again. )
+
+```
+http://[zeppelin-server]:[zeppelin-port]/api/interpreter/load/[interpreter_group_name]/[interpreter_name]
+```
+The Restful method will be <code>**POST**</code>. And the parameters you need are like:
+
+  1.  **Artifact:** Maven artifact ( groupId:artifactId:version ) 
+
+  2.  **Class Name:** Package name + Interpreter class name
+
+  3. **Repository ( optional ):** Additional maven repository address
+
+For example, if you want to load `markdown` interpreter to your Zeppelin, above parameters and the URL you need may like:
+
+```
+http://127.0.0.1:8080/api/interpreter/load/md/markdown
+```
+
+```
+{
+  "artifact": "org.apache.zeppelin:zeppelin-markdown:0.6.0-incubating-SNAPSHOT",
+  "className": "org.apache.zeppelin.markdown.Markdown",
+  "repository": {
+    "url": "http://dl.bintray.com/spark-packages/maven",
+    "snapshot": false
+  }
+}
+```
+
+
+The meaning of each parameters is: 
+
+  1. **Artifact**
+	- groupId: org.apache.zeppelin
+	- artifactId: zeppelin-markdown
+	- version: 0.6.0-incubating-SNAPSHOT
+
+  2. **Class Name**
+	- Package Name: org.apache.zeppelin
+	- Interpreter Class Name: markdown.Markdown
+
+  3. **Repository ( optional )**
+	- Url: http://dl.bintray.com/spark-packages/maven
+	- Snapshot: false
+
+> <b>Please note: </b>The interpreters you downloaded need to be **reload**, when your Zeppelin server is down. 
+
+### 2. Unload
+If you want to **unload** the interpreters using REST API, 
+
+```
+http://[zeppelin-server]:[zeppelin-port]/api/interpreter/unload/[interpreter_group_name]/[interpreter_name]
+```
+In this time, the Restful method will be <code>**DELETE**</code>.
+
+<br/>
+## What is the next step after Loading ?
+
+Maybe, you are curious several things about how you can actually use the interpreter in Zeppelin. 
+ 
+### Q1. Where is the location of interpreters you downloaded ?
+ 	
+Actually, the answer about this question is in the above picture. Once the REST API is called, the `.jar` files of interpreters you get are saved under `ZEPPELIN_HOME/local-repo` first. Then, they will be copied to `ZEPPELIN_HOME/interpreter` directory. So, please checkout your `ZEPPELIN_HOME/interpreter`.
+
+### Q2. Then, how can I use this interpreter ?
+ 
+After loading, you can create and configure the interpreter at the Zeppelin **Interpreter tab**. Then, what is the next ? Please follow these simple steps. It will be really straightforward. 
```

Comment:
You could probably Squeeze it into:
After loading an interpreter, you can use it by creating and configuring it in Zeppelin's **Interpreter tab**.